packages/common/lib/types/Generic.types.ts

```diff
@@ -202,5 +202,7 @@ export interface EndpointMetadata {
   token_endpoint: string;
   credential_endpoint: string;
   authorization_endpoint?: string;
+  pushed_authorization_request_endpoint?: string;
+  require_pushed_authorization_endpoint?: boolean;
```

Comment:
I think I asked this already, but which spec does this value come from? This is a payload. We cannot makeup something ourselves here (if that is the case)

Didn't you happen to mean require_pushed_authorization_requests from https://www.rfc-editor.org/rfc/rfc9126#section-6

Reply:
It's from [here](https://www.rfc-editor.org/rfc/rfc9126#name-authorization-server-metada) actually and it indicates whether the authorization server accepts authorization request data only via PAR.

Reply:
It's from [section 5](https://www.rfc-editor.org/rfc/rfc9126#name-authorization-server-metada). It indicates whether the authorization server accepts authorization request data only via PAR

Reply:
Values should not be in this object to begin with. This is an object we create based on retrieval of several metadata endpoints. The value should be in the OAuth2 Server metadata, so also not in the IssuerMetadata. Will fix that